learntools/core/problem.py

```diff
@@ -68,17 +68,9 @@ def check_whether_attempted(self, *args):
 
 class ThoughtExperiment(Problem):
 
-    # By default, ThoughtExperiment subclasses have no bearing on progress tracking.
-    _counts_for_points = False
```

Comment:
The _counts_for_points property no longer has been deprecated, and this line should be removed no matter what. It has no effect any more.